src/libsystemd/libsystemd.sym

```diff
@@ -546,4 +546,6 @@ global:
         sd_bus_set_sender;
         sd_bus_get_sender;
         sd_bus_message_set_sender;
+        sd_event_source_get_io_fd_own;
+        sd_event_source_set_io_fd_own;
```

Comment:
These should go on LIBSYSTEMD_237 as 236 was already released

Reply:
@fsateler look closely, that's where it already is. (yay, the chaining up is confusing)

Reply:
Arrrgh. sorry for the noise.